docs/content/en/docs/installation.md

```diff
@@ -32,18 +32,4 @@ To make use of the package, you need a running instance of GoodData. If you do n
 
 ### Versioning
 
-The Python SDK is versioned. The version you should be using depends on the type of GoodData distribution you are using.
-
-#### GoodData Cloud
-
-We recommend you always work with the newest (supported) Python available.
-
-#### GoodData.CN
-
-Here is a mapping table of compatibility, which version of Python SDK to use with each version of GoodData.CN
-
-| Gooddata.CN | Python SDK  |
-|---|---|
-| v2.5 | v1.5 |
-| v2.4 | v1.4 |
-| v2.3 | v1.3 |
+The Python SDK is versioned. When working with GoodData Cloud, we recommend you always work with the newest (supported) Python available.
```

Comment:
I am still determining the wording. As I read it, users should use the newest version of Python. Did you want to say to use the latest version of Python SDK?